client/package.json

```diff
@@ -30,5 +30,11 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
```

Comment:
please install  `eslint-config-prettier` 
to turn off all ESLint rules that are unnecessary or might conflict with Prettier rules.

Reply:
I installed it in the dependencies is that ok or should I switch it  ??

Reply:
I have changed it from dependencies into dev ...Done